### PR TITLE
markdown text formatting

### PR DESCRIPTION
**Pull request by heiderich**
*foo*
**foo**
***foo***

### CODE REVIEW

File: UltiSnips/markdown.snippets [hunk 1]
```diff
@@ -50,6 +50,23 @@ snippet spar "Paragraph" b
...
+
+snippet * "cursive"
+*${1:${VISUAL}}*$0
+endsnippet
+
+snippet ** "bold"
+**${1:${VISUAL}}**$0
+endsnippet
+
+snippet *** "bold"
```
Comment (lpil) [inline UltiSnips/markdown.snippets]: Is this text right?

File: UltiSnips/markdown.snippets [hunk 2]
```diff
@@ -50,6 +50,23 @@ snippet spar "Paragraph" b
 $0
 endsnippet
 
+###################
+# Text formatting #
+###################
+
+snippet * "cursive"
```
Comment (lpil) [inline UltiSnips/markdown.snippets]: Would a letter based trigger be better? These are normally easier to type.

Would "italics" be a better description? It doesn't output cursive text.

Comment (heiderich) [reply UltiSnips/markdown.snippets]: I like the asterisk(s) as a trigger. This way the user does not need to remember another trigger.

I agree. "italics" would be a better description. I corrected it in b8f54d395e2f0b20b0ae12e3e21dddad2a0b3fda.

It seems that the concrete formatting is indeed not specified in the commonmark specification. However the descriptions I used seem to describe well the concrete "effect" of the asterisks in common implementations (at least those I used).

File: UltiSnips/markdown.snippets (hunk 1 above)
Comment (heiderich) [reply UltiSnips/markdown.snippets]: No, this was a copy & paste error. I corrected it in b8f54d395e2f0b20b0ae12e3e21dddad2a0b3fda.

**Pull request merged**